rules/aip0158/response_repeated_first_field_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0158
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestResponseRepeatedFirstField(t *testing.T) {
+	tests := []struct {
+		testName    string
+		FirstField  string
+		SecondField string
+		problems    testutils.Problems
+	}{
+		{
+			"Valid",
+			"repeated Book books = 1;",
+			"string next_page_token = 2;",
+			testutils.Problems{}},
+		{
+			// should have at least 1 field
+			"InvalidType",
+			"",
+			"",
+			testutils.Problems{{}}},
+		{
+			// first field by number should be repeated.
+			"InvalidType",
+			"repeated string next_page_token = 2;",
+			"Book books = 1;",
+			testutils.Problems{{}}},
```

Comment:
Should we expect a finding/problem here?

Reply:
Yes, we expect a problem here. The problem detail is added in line 68-74.

Reply:
(Sorry for the late response) I suggest you to remove the `problems` field in the test cases, as it provides no information about the test cases.

Reply:
The `problems` field is used. It is set at line 69.

Reply:
```suggestion
			testutils.Problems{{Message: ""be repeated"}}},
```